### PR TITLE
Update manage-migrations.md

### PR DESCRIPTION
**Pull request by lucasdigital**
The first draft for DOCU-1268, to be merged into master, as agreed.

### CODE REVIEW

File: docs/manage-migrations.md [hunk 1]
```diff
@@ -169,6 +169,20 @@ The **Failed Operations** panel on LiveData UI's Migration screen provides a cou
   java.io.IOException: Error rescanning [/km/gold/edf_gold/icore_order_data/data_dt=20210301/part-m-2839] [not found]
   ```
 
+#### Filter and Search Failed Operations
```
Comment (AshVic) [inline docs/manage-migrations.md:172]: Other headers in this topic use sentence style so would suggest for consistency; 
"Filter and search for failed operations" (with the addition of the word "for")

File: docs/manage-migrations.md [hunk 2]
```diff
@@ -169,6 +169,20 @@ The **Failed Operations** panel on LiveData UI's Migration screen provides a cou
   java.io.IOException: Error rescanning [/km/gold/edf_gold/icore_order_data/data_dt=20210301/part-m-2839] [not found]
   ```
 
+#### Filter and Search Failed Operations
+
+Filter or search for failed operations by using the *Operation type* drop-down selector on the Migration Check Report panel. The filter contains the following operations:
```
Comment (AshVic) [inline docs/manage-migrations.md:174]: I am not sure if we are to use italics for this "Operation type". It looks like our style guide states we should use bold font.
https://workspace.wandisco.com/pages/viewpage.action?pageId=63733822
Therefore, we should use bold for the following:
- Operation type (if this is how it is written on the UI) - I can only see the wireframe which seems to call it "Operation"?
- Migration Check Report (if that is what appears on the UI) - the wireframe calls it "Migration Check" - sorry I can't access the current UI.

Another style point is that the style guide references the MS style guide and the term used is "dropdown" (without selector): https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dropdown

Final style note: I would address the user directly and say "You can filter by the following operations:"

File: docs/manage-migrations.md [hunk 3]
```diff
@@ -169,6 +169,20 @@ The **Failed Operations** panel on LiveData UI's Migration screen provides a cou
...
 
+#### Filter and Search Failed Operations
+
+Filter or search for failed operations by using the *Operation type* drop-down selector on the Migration Check Report panel. The filter contains the following operations:
+
+| Operation | Description |
+| ------ | ----------- |
+| **Create** | Create operations set up a new data migration. For more information, see [`Create`](create-migration.md).  |
+| **Migration (Transfer file)**   | Migration operations handle the transfer of data to be migrated.  |
+| **Unlink (Delete Path)** | Unlink operations remove files under migration. |
```
Comment (AshVic) [inline docs/manage-migrations.md:180]: Line 179 - the second word in parentheses is lower case but this line has title case? Maybe change for consistency to "Delete **p**ath"?

**Review by AshVic — CHANGES_REQUESTED** [review]
Few wee changes

File: docs/manage-migrations.md (hunk 2 above)
Comment (lucasdigital) [reply docs/manage-migrations.md:174]: Oops, yes for "Operation type". Updating with all points.

File: docs/manage-migrations.md (hunk 3 above)
Comment (lucasdigital) [reply docs/manage-migrations.md:180]: Agreed! I'll double-check that these are consistent with the latest version. The version I referenced used this pattern except for **Metadata (Change Permissions)**.

**Review by AshVic — APPROVED** [review]
LGTM!

**Pull request merged**